src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -54,6 +55,14 @@ public ActionForward route(ActionMapping mapping, ActionForm form, HttpServletRe
         return forward;
     }
     
+    @Override
+    public ActionForward copy(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response) throws Exception {
+    	RecurringDisbursementVoucherForm recurringForm = (RecurringDisbursementVoucherForm) form;
+    	recurringForm.getPdpStatuses().clear();
+    	ActionForward forward = super.copy(mapping, form, request, response);
```

Comment:
There is nothing in the super.copy() that will alter the pdpStatuses list but I can move that after just to be cautious.

The recurringDisbursementVoucherDetails are on the RCDV document and the list is reset on the copy method of the document so there does not seem to be a need to loop over this list and reset the DV doc number. Am I looking at the wrong list?